CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+# 17.0.1 (January 26, 2021)
+- Fixed pgdocker directory permissions issue with Local Docker installer: https://github.com/ansible/awx/pull/9152
```

Comment:
> and will also include a UI fix.

What's the UI fix?

Reply:
@unlikelyzero do you have a link to the issue you mentioned?

Reply:
https://github.com/ansible/awx/pull/9093

Reply:
This just got merged, I updated the changelog and rebased.